portfolio/src/main/webapp/index.html

```diff
@@ -10,9 +10,26 @@
     <div id="content">
       <h1>My Portfolio</h1>
       <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+	<img src="images/profile.jpg"></img>
```

Comment:
no need to close the img tag: `<img src="images/profile.jpg">` is enough (img tag is special case where you don't need to close it).